docs/0-zarf-overview.md

```diff
@@ -159,39 +159,77 @@ In the more complex use case, your package consists of updates for many apps / s
 
 ## Quick Start
 
-:::info
+1.  Select your systems OS below
+2.  Ensure you have the pre-requisite applications running
+3.  Enter the commands into your terminal
+
+<Tabs>
+<TabItem value="macOS" label="macOS" default>
+  
+ :::info
 
-This quick start requires you to already have [home brew](https://brew.sh/) package manager installed on your machine.
+This quick start requires you to already have 
+- [home brew](https://brew.sh/) package manager installed on your machine.
+- [Docker](https://www.docker.com/) installed and running on your machine
```

Comment:
Might be good to mention you don't need a K8s cluster or docker, but for this quick demo we will use it.